src/components/Modals/CourseMenu.vue

```diff
@@ -40,8 +41,15 @@
 
 <script>
 export default {
+  props: {
+    semId: Number,
+    isCompact: Boolean
+  },
   data() {
     return {
+      isA: this.semId % 2 === 0 && !this.isCompact,
```

Comment:
The problem with this approach is that depending on the screen size, compact view might have less than 4 semesters in a row, and the regular view might have only 1 semester in a row. Need to resolve this once all of the color issues are fixed with designers, so do not worry about this for now.